config/v1/types_operatorhub.go

```diff
@@ -0,0 +1,67 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OperatorHubSpec defines the desired state of OperatorHub
+type OperatorHubSpec struct {
+	// hubSources is the list of default OperatorSources and their configuration
+	HubSources []HubSource `json:"hubSources,omitempty"`
```

Comment:
What does it mean when hubSources is empty?

Reply:
The marketplace has a notion of [default OperatorSources](https://github.com/operator-framework/operator-marketplace/tree/master/defaults) that it manages. if `HubSources` is empty it indicates to the operator that these `OperatorSources` need to be present on the cluster and it ensures that they are always present.